examples/github-deploy-only.yml

```
@@ -0,0 +1,82 @@
+name: Build and upload to PyPI
+
+on: [push, pull_request]
+
+jobs:
+  build_wheels:
+    name: Build wheels on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-18.04, windows-latest, macos-latest]
+
+    steps:
+    - uses: actions/checkout@v1
+
+    - uses: actions/setup-python@v1
+      name: Install Python
+      with:
+        python-version: '3.7'
+
+    - name: Install cibuildwheel
+      run: |
+        python -m pip install cibuildwheel==1.4.2
+
+    - name: Install Visual C++ for Python 2.7
+      if: startsWith(matrix.os, 'windows')
+      run: |
+        choco install vcpython27 -f -y
+
+    - name: Build wheels
+      run: |
+        python -m cibuildwheel --output-dir wheelhouse
+
+    - uses: actions/upload-artifact@v1
```

nit: for a couple of these actions, a `@v2` is available and recommended.

---

At some point, there was a discussion: https://github.com/joerick/cibuildwheel/pull/194#discussion_r390019467

But feel free to update the ones that can be updated. It's your branch after all :-D But then we should also do this in the other example, and in cibuildwheels own CI configuration.

---

Python v2 is easy, it just allows patch version pinning, which we don't use. You should be using upload/download v2, as they are simpler, will automatically merge, and don't even require a name (`artifact` is the default). Checkout v2 is the hardest, as it didn't support tag history at all until a week ago, and only supports tag history if you do a `fetch-depth: 0` checkout. If you use v2, `setuptools_scm` becomes trickier to use because of that. So not sure if that should change as the example yet. There's an ongoing discussion about how this could be improved here https://github.com/actions/checkout/issues/249.

---

```suggestion
    - uses: actions/upload-artifact@v2
```
You should also drop the `name: wheels`, since this does not match the download below.

---

Done! Thanks both :-)